test/delayed_queue_test.rb

```diff
@@ -1,6 +1,14 @@
 # vim:fileencoding=utf-8
 require_relative 'test_helper'
 
+def assert_resque_key_exists?(key)
+  if Gem::Requirement.create('< 4').satisfied_by?(Gem::Version.create(Redis::VERSION))
+    assert(!Resque.redis.exists(key))
+  else
+    assert(!Resque.redis.exists?(key))
+  end
+end
```

Comment:
Nice. This was around the signature change for `exists?`, right? https://github.com/redis/redis-rb/blob/master/CHANGELOG.md#420